src/domains/wallet/pages/WalletDetails/components/WalletHeader/WalletHeader.tsx

```diff
@@ -255,7 +252,7 @@ export const WalletHeader = ({
 						<Avatar size="lg" address={wallet.address()} shadowClassName="ring-theme-secondary-900" />
 					</div>
 
-					<div className="flex flex-col w-full">
+					<div className="flex flex-col w-full overflow-hidden">
```

Comment:
this destroys its accessible UI
![Screenshot 2021-06-08 153430](https://user-images.githubusercontent.com/24996341/121175044-e9335780-c85a-11eb-9c0d-ae7137b5b920.png)

```suggestion
					<div className="flex flex-col w-full">
```

Reply:
ed6d441

Reply:
![Screenshot 2021-06-08 180748](https://user-images.githubusercontent.com/24996341/121198374-5f8f8400-c872-11eb-94bb-12f6b5d940ae.png)
and aren't align